src/sqlitedb/database.hpp

```diff
@@ -26,9 +26,9 @@ namespace db
   template <size_t N>
   constexpr bool is_cstr<const char[N]> = true;
```

Comment:
could you also add `inline` to this `constexpr` while you are here.

Reply:
Templates are always implicitly inline.  The ones that are explicitly `inline` here are the full template specializations, which aren't.